compile.sh

```diff
@@ -0,0 +1 @@
+mvn clean package -DskipTests
```

Comment:
Is this file used anywhere?

Reply:
This file isn't being used. It's meant as a convenience script for developers. Should we remove it?

Reply:
This is a pretty small convenience script. I think if anything, we should just add a note to the README about packaging up the app as a jar. I'm not sure that's really needed anyway though, since this is designed to run in a docker container.

Reply:
That's fair! This file has been removed.